core/math/expression.h

```diff
@@ -73,9 +73,10 @@ class Expression : public Reference {
 		MATH_MOVE_TOWARD,
 		MATH_DECTIME,
 		MATH_RANDOMIZE,
-		MATH_RAND,
+		MATH_RANDI,
 		MATH_RANDF,
-		MATH_RANDOM,
+		MATH_RANDF_RANGE,
+		MATH_RANDI_RANGE,
```

Comment:
If you're up for one last refactoring, it would be good to use the opportunity to change the order here:
```
MATH_RANDI
MATH_RANDF
MATH_RANDI_RANGE
MATH_RANDF_RANGE
```
(i.e. int first, then float for the ranged version too)

And then adapt the order in all places where these enums are used to keep the same order as the enum.

Reply:
Or the other way around (float, then int) if we want alphabetical order instead:
```
MATH_RANDF
MATH_RANDI
MATH_RANDF_RANGE
MATH_RANDI_RANGE
```